.travis.yml

```diff
@@ -43,7 +43,7 @@ env:
     - PATH=$USERDIR/bin:$PATH
     - LD_LIBRARY_PATH=$USERDIR/lib:$LD_LIBRARY_PATH
     - LEATHERMAN_VERSION=0.9.0
-    - CPPHOCON_VERSION=0.1.2
+    - CPPHOCON_VERSION=0.1.3
```

Comment:
I'm not sure, given that the rest of the functionality in this PR doesn't work at all without it. What do you think? Do we usually always split it out?